packages/client/src/products/types/productGrouping.types.ts

```diff
@@ -0,0 +1,31 @@
+export type DigitalAsset = {
+  mediaType: string;
+  displayOrder: number;
+  size: string;
+  url: string;
+  type: number;
```

Comment:
Could this be an enum? According to the api documentation:
```
DigitalAssetType

0 - Default
1 - ItemSwatch
```

Reply:
Nice, I created an enum for it, thanks!